pkg/kubectl-explain/slide-in.js

```diff
@@ -0,0 +1,51 @@
+import Vue from 'vue';
+
+import Panel from './components/SlideInPanel';
+import { OpenAPI } from './open-api';
+
+const PANEL_ID = 'kubectl-explain';
+
+// Cache of the Open API Data for a cluster
+const openAPI = new OpenAPI();
+
+// Slide in panel component
+let slideInPanel;
+
+/**
+ * Show the slide-in panel with the resource explanation
+ */
+export async function explain(store, route) {
+  const typeName = route.params?.resource;
+  const schema = typeName ? store.getters[`cluster/schemaFor`](typeName) : undefined;
+  const cluster = store.getters['currentCluster'];
```

Comment:
as per top level review comment, i think this can be `const cluster = store.getters['currentCluster']; || 'local'`

Reply:
Fixed